FND/Places/VirtualPlaces.rdf

```diff
@@ -49,13 +49,36 @@
 		<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/"/>
 		<owl:imports rdf:resource="https://www.omg.org/spec/LCC/Countries/CountryRepresentation/"/>
 		<owl:imports rdf:resource="https://www.omg.org/spec/LCC/Languages/LanguageRepresentation/"/>
-		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/20200201/Places/VirtualPlaces/"/>
+		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/20200301/Places/VirtualPlaces/"/>
 		<skos:changeNote>The http://www.omg.org/spec/EDMC-FIBO/FND/20141101/Places/VirtualPlaces.rdf version of this ontology was modified for the FIBO 2.0 RFC to integrate it with LCC.</skos:changeNote>
-		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20180801/Places/VirtualPlaces.rdf version of this ontology was modified to eliminate duplication of concepts in LCC.</skos:changeNote>
+		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20180801/Places/VirtualPlaces.rdf version of this ontology was modified to eliminate duplication of concepts in LCC and email address and telephone number.</skos:changeNote>
 		<skos:changeNote>This ontology was added to Foundations in advance of the December 2014 Long Beach meeting in support of the SEC specification.</skos:changeNote>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Release"/>
 	</owl:Ontology>
 	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-plc-adr;hasElectronicMailAddress">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-plc-adr;hasAddress"/>
+		<rdfs:label>has electronic mail address</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-fnd-plc-vrt;ElectronicMailAddress"/>
+		<skos:definition>specifies an electronic mailbox at which some entity may be located or contacted or may receive correspondence, typically via an Simple Mail Transfer Protocol (SMTP) based communications system</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-plc-adr;hasTelephoneNumber">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-plc-adr;hasAddress"/>
+		<rdfs:label>has telephone number</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-fnd-plc-vrt;TelephoneNumber"/>
+		<skos:definition>indicates a virtual address composed of a sequence of digits assigned to a fixed-line telephone subscriber station, a wireless electronic telephony device, such as a radio telephone or a mobile telephone, or other similar device</skos:definition>
```

Comment:
This is the definition of a telephone number based on the CCITT signaling protocol.  It's really difficult to be precise about something and relax the terminology.  WF uses similar terminology in their enterprise glossary, fyi.  If you look it up in the protocol definition, you'll see that this is essentially what a telephone number is.  Then we can subclass it for mobile telephony, facsimile, various more modern protocols and its still correct.

Reply:
I still think we should provide another alternative to a "device" of some kind.

Reply:
Revised to say 'that may be assigned to' rather than 'assigned to', and at the end, changed 'or other similar device' to 'or other similar device or service'.